packages/server/graphql/intranetSchema/mutations/updateOAuthRefreshTokens.ts

```diff
@@ -0,0 +1,27 @@
+import {GraphQLInt} from 'graphql'
+import {requireSU} from '../../../utils/authorization'
+import {GQLContext} from '../../graphql'
+import getAtlassianAuthsToUpdate from '../../../postgres/queries/getAtlassianAuthsToUpdate'
+import {Threshold} from '~/types/constEnums'
+import {isNotNull} from '../../../utils/predicates'
+
+const updateOAuthRefreshTokens = {
+  type: GraphQLInt,
+  description: `Updates Atlassian OAuth tokens that haven't been updated for more than 14 days`,
+  args: {},
```

Comment:
add
```ts
before : {
  type: GraphQLISO8601
}
```

see `dailyPulse for the pattern

Reply:
Done!